frontend/src/app/signUp/Organization/OrganizationForm.test.tsx

```diff
@@ -79,7 +79,7 @@ describe("OrganizationForm", () => {
 
   it("displays form errors when submitting invalid input", async () => {
     const { user } = renderWithUser();
-    await fillInDropDown(user, getOrgStateDropdown(), "VI");
+    await fillInDropDown(user, getOrgStateDropdown(), "AS");
```

Comment:
how much work would it be to replace this with something that won't ever fail once a new jurisdiction comes online? if it's not too much effort, it might be better in the long run to assert on something obviously fantastical (Asgard? Middle Earth? Insert other favorite mystical place here? 😛 ) than somewhere that might get activated in the future 

we can handle this in a follow up PR though to get the change in

Reply:
With the way the test is currently written, the value that it is negatively testing for still needs to be populated in the org state dropdown list. Unless we significantly change this test or remove it, I think it'll ultimately be faster and simpler for us to just update it with a different offline jurisdiction as necessary